pkgs/top-level/all-packages.nix

```diff
@@ -5095,7 +5095,7 @@ with pkgs;
 
   texmaker = qt6Packages.callPackage ../applications/editors/texmaker { };
 
-  texstudio = qt6Packages.callPackage ../applications/editors/texstudio { };
+  texstudio = qt6Packages.callPackage ../by-name/te/texstudio/package.nix { };
```

Comment:
this line can be removed all by-name packages get included automatically

Reply:
Even though the package is using `qt6Packages.callPackage`?

Reply:
Ah, sorry I didn't spot qt6Packages.callPackage, if it's easier you can revert to previous version